File: src/Goods.Domain/Products/ProductCategory.cs
using System.ComponentModel.DataAnnotations;

namespace Goods.Domain.Products;

public enum ProductCategory
{
	[Display(Name = "Фрукты")]
	Fruits = 1,

	[Display(Name = "Овощи")]
	Vegetables = 2,

	[Display(Name = "Молочные продукты")]
	Dairy = 3,

	[Display(Name = "Мясо")]
	Meat = 4,

	[Display(Name = "Рыба и морепродукты")]
	Fish = 5,

	[Display(Name = "Хлебобулочные изделия")]
	Bakery = 6,

	[Display(Name = "Крупы и злаки")]
	Cereals = 7,
}

File: src/Goods.BackOffice/client-app/src/domain/products/productCategory.ts
export enum ProductCategory {
	Fruits = 1,
	Vegetables = 2,
	Dairy = 3,
	Meat = 4,
	Fish = 5,
	Bakery = 6,
	Cereals = 7
}

export namespace ProductCategory {
	export const getDisplayName = (category: ProductCategory): string => {
		switch (category) {
			case ProductCategory.Fruits:
				return 'Фрукты';
			case ProductCategory.Vegetables:
				return 'Овощи';
			case ProductCategory.Dairy:
				return 'Молочные продукты';
			case ProductCategory.Meat:
				return 'Мясо';
			case ProductCategory.Fish:
				return 'Рыба и морепродукты';
			case ProductCategory.Bakery:
				return 'Хлебобулочные изделия';
			case ProductCategory.Cereals:
				return 'Крупы и злаки';
		}
	};
}
